common_types/src/datum.rs

```diff
@@ -4,6 +4,7 @@
 
 use std::{convert::TryFrom, fmt, str};
 
+#[cfg(feature = "arrow_deps")]
```

Comment:
Now the cfg statements are distributed across too many places in this mod and maybe we can declare a new sub-mod called `arrow_convert` in this file into which all the methods and imports relavant to `arrow_deps` can be grouped. After that we can use only one `#[cfg(feature = "arrow_deps")]` to control the mod `arrow_convert`.

And I guess this parttern can be applied to other mods, such as `column`, `column_schema` and so on.

Reply:
Exactly!